base16ct/src/lib.rs

```diff
@@ -0,0 +1,228 @@
+#![no_std]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![forbid(unsafe_code)]
+#![warn(missing_docs, rust_2018_idioms)]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_root_url = "https://docs.rs/base16ct/0.0.0"
+)]
+#![doc = include_str!("../README.md")]
```

Comment:
I am not sure it's worth to include the whole readme while it does not contain any code examples. If anything, everything in it except the first 3 paragraphs looks like an unnecessary clutter to me in the docs context.